### PR TITLE
Add close status notification

### PR DESCRIPTION
**Pull request by CGastrell**
This PR adds a notice at the bottom of the popover to indicate if the Feedback block is closed or its expiry date has passed

## Test instructions
Checkout and `make client`. On a Feedback block, change settings to "Closed" or "Closed after" and a past date. The notice should be noticeable

### CODE REVIEW

**Review by digitalwaveride — CHANGES_REQUESTED** [review]
Thanks Christian for bringing this in so fast!!! 🙏 

Works for me!
Just some styling fine-tuning would be great. Looking at this right now... 

image



- The color of that bar is a bit an issue as it is squeezed between the two buttons. We could use the same color as the buttons, but than it would maybe not stand out enough.
But better would be maybe to go with neutral standard Gutenberg "dark color" that should work with most Submit button colors etc: 
background-color: #1e1e1e

- For the text-styling, could we go with:

```
font-size: 14px;
line-height: 150%;
padding: 6px;
```

- The Text
I would keep it a tiny bit simpler, and mimic the same statement that we have on the poll block for that closed state:

`This Feedback Form is Closed`

All together could like this:

image

File: client/blocks/feedback/edit.js [hunk 1]
```diff
@@ -200,6 +201,12 @@ const EditFeedbackBlock = ( props ) => {
 		height,
 	};
 
+	const isCurrentlyClosed =
```
Comment (ice9js) [inline client/blocks/feedback/edit.js:204]: Nit: Not a huge fan of overly descriptive variable names, let's maybe cut it down to `isClosed`?

File: client/components/feedback/style.scss [hunk 2]
```diff
@@ -4,6 +4,11 @@
 	display: flex;
 	position: fixed;
 	z-index: 100;
+
+	> div:last-child:not(.crowdsignal-forms-feedback__trigger) {
```
Comment (ice9js) [inline client/components/feedback/style.scss:8]: What is this and why can't we just give it a proper class name?

**Review by ice9js — COMMENTED** [review]
Overall 👍  I left two small comments.

**Review by digitalwaveride — APPROVED** [review]
😍  🚀

File: client/components/feedback/style.scss (hunk 2 above)
Comment (CGastrell) [reply client/components/feedback/style.scss:8]: We need the rounded corners at the bottom of the modal, and the new notice that goes there is just appended to the popover. So, whatever div is last, excluding the `trigger`, should have this roundness.

The notice does have a class `.crowdsignal-forms-feedback__closed-notice`, but if we just put the rounded corners on both it doesn't look good, like a double smile on a face.

**Pull request merged**